### DISCUSSION TITLE
openEMS simulations in Keysight Labs video

### DISCUSSION THREAD

**Discussion by samerps** [thliebig/openEMS-Project#45] [Show and tell]
Keysight Labs recently published a video and did a livestream about PCB transmission lines. There are a few 3D wave animated simulations in those videos to show how a wave travels in microstrips with different widths and with vias. I created these animations and I used Blender to render the data generated from openEMS. 
![image](https://user-images.githubusercontent.com/68449711/225648012-a063b72e-1d50-416f-80f1-e76cf9bb47ed.png)

You can watch the video on Youtube [https://www.youtube.com/watch?v=jqAiKnjjDE8](https://www.youtube.com/watch?v=jqAiKnjjDE8)
The openEMS animations appear at 1:40, 3.15, 7:55

There was also a pre-recorded livestream  [https://www.youtube.com/watch?v=NvLb2xYIens](https://www.youtube.com/watch?v=NvLb2xYIens)
The openEMS animations appear at 7:15, 15:55, 33:05, 37:55

I am planning to create more animations in Blender with openEMS data, I will be publishing them soon here and, on my socials, [Artstation ](https://www.artstation.com/high_voltage), [Twitter](https://twitter.com/samerps), [LinkedIn](https://www.linkedin.com/in/samerps/), have a look at some my previous published work. 

Thanks to Thorsten and developers for creating openEMS!

**Comment by 0xCoto** [comment]
Really cool stuff, great work! Would love to see a tutorial on how you'd import your data into blender and make an animation like this. :)

**Comment by thliebig** [comment]
Yes looks great, thanks for the links and image.
